### ISSUE TITLE
Doesn't build when SSLv3 is disabled in OpenSSL

### ISSUE THREAD

**Issue by peikk0** [lefcha/imapfilter#114]
Imapfilter doesn't build when OpenSSL is compiled without support for SSLv3:
```
--- imapfilter ---
cc -o imapfilter  -L/usr/local/lib -L/usr/local/lib -Wl,-rpath,/usr/local/lib -fstack-protector auth.o buffer.o cert.o core.o file.o imapfilter.o list.o log.o lua.o  memory.o misc.o namespace.o pcre.o regexp.o request.o response.o  session.o signal.o socket.o system.o -lm -llua-5.2 -lpcre -lssl -lcrypto 
imapfilter.o: In function `main':
imapfilter.c:(.text+0x1cb): undefined reference to `SSLv3_client_method'
cc: error: linker command failed with exit code 1 (use -v to see invocation)
*** [imapfilter] Error code 1
```

It probably doesn't build with LibreSSL either (no SSLv3 support at all).

**Comment by peikk0** [comment]
A patch can be found here: https://github.com/Sp1l/freebsd-ports/blob/master/mail/imapfilter/files/patch-src_imapfilter.c

**Comment by lefcha** [comment]
Thanks a lot, I just fixed this problem!

**Comment by lefcha** [comment]
Closed by c470c45d1fcb9bf1d51e9ffba6c7be9186dcc6d2.

**Comment by peikk0** [comment]
Thanks! Can you tag a new release with this fix please?

**Comment by lefcha** [comment]
Sure, just released 2.6.5.

**Comment by peikk0** [comment]
Thanks!
